tools/staticbuild/build_lib.sh

```diff
@@ -48,12 +47,6 @@ if [[ $VARIANT == *mkl ]]; then
     cp 3rdparty/mkldnn/build/install/lib/$IOMP_LIBFILE lib
     cp 3rdparty/mkldnn/build/install/lib/$MKLML_LIBFILE lib
     cp 3rdparty/mkldnn/build/install/lib/$MKLDNN_LIBFILE lib
-    cp 3rdparty/mkldnn/build/install/$MKLDNN_LICENSE lib
```

Comment:
Do we not need to copy the mkl-dnn and cub licenses anymore?
I couldn't find them in LICENSE.binary.dependencies.

Also, these changes have broken the `pip/setup.py`. I will post a PR and add you as reviewer =)

Reply:
We are also including the contents of the source LICENSE (LICENSE in the main folder) as well which contains both mkldnn and cub